crates/nu-cli/src/commands/date/command.rs

```diff
@@ -16,7 +16,7 @@ impl WholeStreamCommand for Command {
     }
 
     fn usage(&self) -> &str {
-        "Work with dates."
+        "Apply date function"
```

Comment:
Keeping this consistent with other high level commands like `str`, e.g. `Apply string function`